### PR TITLE
test(#2085): Test repetition at night

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
I've added a GitHub workflow to detect flaky tests and concurrency problems.
Closes: #2085

---

## PR-Codex overview
This PR adds a GitHub action that runs load testing on the `eo-parser`, `eo-maven-plugin`, and `eo-runtime` components every day at 01:30 UTC, using a script that executes Maven tests multiple times to check for flaky tests and concurrency issues.

### Detailed summary
- Added a GitHub action to run load testing on `eo-parser`, `eo-maven-plugin`, and `eo-runtime` components every day at 01:30 UTC
- Added a script that runs Maven tests multiple times to check for flaky tests and concurrency issues
- Script takes command-line options `--max` and `--folder` to specify the number of iterations and the path to the testable module
- Added a to-do item to create a GitHub issue when any of the tests fail

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by Graur — APPROVED** [review]
@volodya-lombrozo Looks good to me
@yegor256 Please, have a look

**Comment by volodya-lombrozo** [comment]
@yegor256 Could you merge that changes, please?

**Comment by volodya-lombrozo** [comment]
@yegor256 Reminder

**Comment by yegor256** [comment]
@volodya-lombrozo I really don't like it when repositories have `scripts/` directory. Maybe it's better to put this code into `src/test/scripts/`?

**Comment by volodya-lombrozo** [comment]
@yegor256 Done. May I ask why it's preferable to place them under `src/test/scripts` rather than just `scripts`? Is it a common practice?

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@volodya-lombrozo actually, a common practice now is to use `scripts/` directory, but I don't like it, because "just scripts" is a poor naming strategy. By having a global directory `scripts/` we are inviting programmers to put any script there, no matter what they are about: testing, deploying, packaging, configuring, etc. If we encourage this behaviour, very soon we will have many script there, with no logical structure.

File: .github/workflows/daily.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  schedule:
+    # Run load testing at 01:30 UTC every day
+    - cron: '30 1 * * *'
+jobs:
+  build:
+    runs-on: [ ubuntu-latest, macos-latest ]
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          SCRIP="${GITHUB_WORKSPACE}/src/test/scrips/test-repetition.sh"
```
Comment (yegor256) [inline .github/workflows/daily.yml:12]: @volodya-lombrozo I believe it's a typo, should be `SCRIPT`

**Comment by yegor256** [comment]
@volodya-lombrozo actually, I don't like the idea of "scripts" at all. In this particular example, can't we put this bash code right into the YAML file for GHA?

**Comment by volodya-lombrozo** [comment]
@yegor256 

> @volodya-lombrozo actually, a common practice now is to use scripts/ directory, but I don't like it, because "just scripts" is a poor naming strategy. By having a global directory scripts/ we are inviting programmers to put any script there, no matter what they are about: testing, deploying, packaging, configuring, etc. If we encourage this behaviour, very soon we will have many script there, with no logical structure.

I see. It makes sense, actually. Thank you!

> @volodya-lombrozo actually, I don't like the idea of "scripts" at all. In this particular example, can't we put this bash code right into the YAML file for GHA?

I want to have a chance to run this scripts during development. I quite often run them when check concurrency issues or flaky tests. If I place the entire script to GHA, I will need to extract it each time when I will need it during development.

**Pull request merged**